js/components/Team/TeamTitle.js

```diff
@@ -0,0 +1,65 @@
+import Component from "../../core/component.js";
+import store from '../../store/index.js';
+import { addTeam ,deleteTeam} from "../../service/TeamApi.js";
+
+export default class TeamTitle extends Component {
+    constructor() {
+        super({
+            store,
+            element: document.querySelector('.team-list-container')
+        });
+    }
+
+    teamTemplate = (teamList) => {
+        const template = teamList.map((team) => {
+            return `<div class="team-card-container">
+                <a href="/kanban.html?name=${team._id}"  class="card">
+                    <div class="card-title">
+                        ${team.name}
+                    </div>
+                </a>            
+              <button class="destroy" data-id="${team._id}">X</button>
+            </div>`
+        }).join('')
+        return template;
+
+    }
+    addTeamTemplate = () => {
+        return `<div class="add-team-button-container">
+            <button id="add-team-button" class="ripple">
+                <span class="material-icons">add</span>
+            </button>
+        </div>`
+    }
+
+    render() {
+        let self = this;
+
+        const template = this.teamTemplate(store.state.team);
+        self.element.innerHTML = template;
+        self.element.innerHTML += this.addTeamTemplate();
```

Comment:
```suggestion
        self.element.innerHTML = template + this.addTeamTemplate();
```
innerHTML은 최대한 적게 실행해주는게 좋습니다! 안그러면 렌더링이 연속으로 일어나요 ㅠㅠ

Reply:
tamplate 문자열을 한번에 만들고  innerHtml을 한번만 쓰면 될까요?

Reply:
넵 맞습니다!